nautobot/ipam/views.py

```diff
@@ -752,6 +752,13 @@ def successful_post(self, request, obj, created, logger):
                     ),
                 )
 
+        # Add IpAddress to interface if interface is in query_params
+        if interface_id := request.GET.get("interface"):
```

Comment:
Seems wrong to me to be checking `request.GET` on a `successful_post` function - should this be part of `request.POST` or something instead?

Reply:
The purpose of this issue is to reintroduce the "Assign IPAddress" button to the Device Interface List page's table.
Clicking the button takes you to the ip-address edit page, and the query param is `interface=uuiid>`.

Reply:
Wonder if it'd be possible and/or more elegant to have the GET to the ip-address edit page render/populate the interface as a form field value (hidden or otherwise) so that here, in the POST handler, it'd be part of the form data instead of a separate query parameter?

Reply:
it means this would
`if interface_id := request.GET.get("interface"):`
to
`if interface_id := request.POST.get("interface"):`
or 
illd modify the `IPAddressForm` to check for `interface`.

Any of the above would require the addition of more code, which makes me wonder if it's worth the effort to archive the same thing?